contents/blog/posthog-vs-hotjar.md

```diff
@@ -0,0 +1,538 @@
+
+### Heatmaps
+
+Heatmaps visualize user activity in your app. They use colors or numbers to show which parts of the page people are interacting with the most. Both Hotjar and PostHog offer heatmaps, although PostHog's offering is more limited:
+
+<div className="overflow-x-auto -mx-5 px-5">
+<table className="w-full mt-4" style="min-width: 600px;">
+    <thead>
+        <tr>
+			<td className="w-3/12"></td>
+            <td><strong>Hotjar</strong></td>
+            <td><strong>PostHog</strong></td>
+            <td></td>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><strong>Click and tap heatmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Show where users are clicking.</td>
+        </tr>
+        <tr>
+            <td><strong>Move heatmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Show where users have moved their mouse on the screen.</td>
+        </tr>
+        <tr>
+            <td><strong>Scroll heatmaps</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td>Show how far down the page your users scroll.</td>
+        </tr>
+        <tr>
+            <td><strong>Rage clicks</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Show where users repeatedly click in a short period of time.</td>
+        </tr>
+        <tr>
+            <td><strong>Combine rage click data with product analytics</strong></td>
+            <td className="text-center"><span className="text-red text-lg">✖</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>View related events and funnel data for users who rage click.</td>
+        </tr>
+        <tr>
+            <td><strong>Heatmap data filters</strong></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td className="text-center"><span className="text-green text-lg">✔</span></td>
+            <td>Filter heatmaps by date, country etc.</td>
+        </tr>
+    </tbody>
+</table>
+</div>
+
+### In-app messages and user research
+
+Hotjar includes feedback widgets, surveys, and an integrated user research tool for gathering feedback from users. PostHog also supports [feedback widgets](/apps/feedback-widget) and [user research scheduling](/apps/user-interviews), although not surveys.
+
```

Comment:
```suggestion

However, because PostHog is entirely open source, it's possible for users to build their own user surveys if needed. 

```

You may want to expand on this, but I think it's good to add. Could arguably be a callout with a `>`